src/gateways/BeaconsGateway.test.ts

```diff
@@ -2,28 +2,31 @@ import axios from "axios";
 import { applicationConfig } from "../config";
 import { IBeacon } from "../entities/IBeacon";
 import { beaconFixture } from "../fixtures/beacons.fixture";
+import { singleBeaconApiResponseFixture } from "../fixtures/singleBeaconApiResponse.fixture";
 import { BeaconsGateway } from "./BeaconsGateway";
+import { IBeaconRequestMapper } from "./mappers/BeaconRequestMapper";
 import { IBeaconResponseMapper } from "./mappers/BeaconResponseMapper";
+import { IBeaconRequest } from "./mappers/IBeaconRequest";
+import { IBeaconResponse } from "./mappers/IBeaconResponse";
 
 jest.mock("axios");
 jest.useFakeTimers();
 
-afterEach(() => {
-  jest.clearAllMocks();
-});
-
 describe("BeaconsGateway", () => {
   const beaconResponseMapper: IBeaconResponseMapper = {
     map: jest.fn(),
   };
 
-  describe("getAllBeacons()", () => {
-    beforeEach(() => {
-      jest.clearAllMocks();
-    });
```

Comment:
See https://github.com/facebook/jest/issues/4000, https://jestjs.io/docs/configuration#clearmocks-boolean and https://jestjs.io/docs/jest-object#jestclearallmocks.

TL;DR - I've set Jest to clear all mocks in between tests by default and removed all `jest.clearAllMocks()` calls from our test suite to improve readability and eliminate mock state bleeding between tests.

Reply:
Good call. Reset state